src/main/java/com/fundacionjala/movies/MovieChildren.java

```diff
@@ -0,0 +1,29 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by Charito on 6/10/2016.
+ */
+public class MovieChildren extends Movie {
+
+    public MovieChildren(String title) {
+        super(title);
+        super.priceRent = 1.5;
+        super.daysAllowed = 3;
+        super.priceDelay = 1.5;
+        super.points = 1;
+    }
+
```

Comment:
I'm not sure that this is the correct form to assign the values of variables inheritance